shared/components/SpeakerListIntro/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const SpeakersListIntro = () => {
+  return (
+    <div>
+      <h1>On Stage</h1>
+      <p>
+        From the community. From the community.
+        From the community. From the community.
+        From the community. From the community.
+        From the community. From the community.
+        From the community.
+      </p>
+    </div>
+  );
+};
```

Comment:
Any reason why this is a separate component? Seems odd to pull it out when it's 5 lines of static HTML.